site/posts/jupyter-notebooks-faq.ipynb

```diff
@@ -108,6 +108,8 @@
     "\n",
     "**A.** As long as your notebook is saved, `ctrl-C` is fine. Otherwise, you will lose the contents of your notebooks that weren’t saved before the shutdown.\n",
     "\n",
+    "To double check that your Jupyter Notebook isn't still running in the background, type `jupyter notebook list` in the terminal to list all active sessions.\n",
+    "\n",
```

Comment:
This works on Macs, but it apparently doesn't work on Windows machines.

Reply:
I think I acknowledged this in the newest push.